app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
+| `_vwo_uuid`                                   | This VWO cookie generates a unique ID for every visitor                                                                                                                                                                                                  | 10 years                                        |
+| `_vwo_ds`                                     | This VWO cookie stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |
+| `_vwo_sn`                                     | This VWO cookie stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |
+| `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | This VWO cookie stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |
+| `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |
+| `_vvwo`                                       | This VWO cookie is a jar of all the VWO cookies. It is currently created only for the Safari browser                                                                                                                                                     | Expires in 2096 years                           |
+
+## Tracking and performance cookies
+
+**Type:** Persistent cookies
+
+**Administered by:** Third parties
+
+**Purpose:** These cookies are used to track information about traffic to the website and how users use the website.  We measure website usage so that we can gain a better understanding of how people use our site. This enables us to improve the information that we provide. It also ensures that the service is available when you want it, and that it delivers information quickly.
+
+The information gathered via these cookies may directly or indirectly identify you as an individual visitor. This is because the information collected is typically linked to a pseudonymous identifier associated with the device you use to access the website. We may also use these cookies to test new pages, features or new functionality of the website to see how people react to them.
```

Comment:
```suggestion
The information gathered via these Cookies may directly or indirectly identify you as an individual visitor. This is because the information collected is typically linked to a pseudonymous identifier associated with the device you use to access the Website. We may also use these cookies to test new pages, features or new functionality of the Website to see how people react to them.
```